files/image_config/ntp/ntp.conf.j2

```diff
@@ -90,32 +138,24 @@ interface listen eth0
 {% endif %}
 interface listen 127.0.0.1
 
-# Access control configuration; see /usr/share/doc/ntp-doc/html/accopt.html for
-# details.  The web page <http://support.ntp.org/bin/view/Support/AccessRestrictions>
-# might also be helpful.
-#
-# Note that "restrict" applies to both servers and clients, so a configuration
-# that might be intended to block requests from certain clients could also end
-# up blocking replies from your own upstream servers.
+{# Access control options -#}
+{% set options = '' -%}
+
+{# Allow additional servers mobilization from the pool. Otherwise we don't need
+that -#}
+{% if is_pools == False -%}
+    {% set options = options ~ ' nopeer' -%}
+{% endif -%}
+{# Disable NTP server functionality. Should stay on when dhcp is enabled -#}
+{# {% if global.server_role == 'disabled' and global.dhcp == 'disabled' -%}
+    {% set options = options ~ ' ignore' -%}
+{% endif -%} #}
 
+# Access control configuration
 # By default, exchange time with everybody, but don't allow configuration.
-restrict -4 default kod notrap nomodify nopeer noquery
-restrict -6 default kod notrap nomodify nopeer noquery
+restrict -4 default kod limited notrap nomodify noquery{{ options }}
+restrict -6 default kod limited notrap nomodify noquery{{ options }}
```

Comment:
Why was `nopeer` removed here? Is it so that in theory, other devices can use this device as a source for the current time?

Reply:
Ah, never mind, I see it manually gets added above.